stablehlo/dialect/StablehloOps.td

```diff
@@ -304,8 +304,9 @@ def StableHLO_CosineOp: StableHLO_UnaryElementwiseOp<"cosine",
 }
 
 def StableHLO_ExpOp: StableHLO_UnaryElementwiseOp<"exponential",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_FpOrComplexTensor> {
-  let summary = "Exponential operator";
+    [Pure, HLO_CompatibleOperandsAndResultType /* exponential_c1 */],
+     HLO_FpOrComplexTensor> {
+  let summary = "Exponential operation";
```

Comment:
Add input constraints as per  the checklist.

Reply:
Exp's all inputs are all from StableHLO_UnaryElementwiseOp, so no clear to add exp_i1 in StableHLO_UnaryElementwiseOp. Same issue for other ops. Please let me know if you think otherwise.

Reply:
```
def StableHLO_ExpOp: StableHLO_UnaryElementwiseOp<"exponential",
    [Pure, HLO_CompatibleOperandsAndResultType /*exponential_c1*/],
     HLO_FpOrComplexTensor /*exponential_i1*/> {
```